src/metabase/cmd/resources/other-env-vars.md

```diff
@@ -221,6 +221,20 @@ This variable affects connections that are severed and undetected by Metabase (t
 
 Unless set otherwise, the default production value for `metabase.query-processor.query-timeout-ms` is used which is 1,200,000 ms (i.e. 1,200 seconds or 20 minutes).
 
+### `MB_JDBC_DATA_WAREHOUSE_DEBUG_UNRETURNED_CONNECTION_STACK_TRACES`
```

Comment:
@jeff-bruemmer let me know if docs look ok (or if I should ping someone else instead)